org.opent2t.sample.thermostat.superpopular/test/package.json

```diff
@@ -10,6 +10,6 @@
   "main": "thermostat.js",
   "license": "MIT",
   "dependencies": {
-    "opent2t": "1.0.1"
+    "opent2t": "*"
```

Comment:
>t": "*" [](start = 11, length = 7)

I just went wtih the wildcard -- but this was one of the tests wanting a specific version which i dont think we want for the unit tests